templates/commands/render_action_test.go

```diff
@@ -224,6 +224,113 @@ func TestWalkAndModify(t *testing.T) {
 	}
 }
 
+func TestWalkAndModify_Seen(t *testing.T) {
+	t.Parallel()
+
+	fooToFooFooVisitor := func(buf []byte) ([]byte, error) {
+		return bytes.ReplaceAll(buf, []byte("foo"), []byte("foofoo")), nil
+	}
+
+	cases := []struct {
+		name            string
+		visitor         walkAndModifyVisitor
+		relPaths        []string
+		seen            map[string]struct{}
+		initialContents map[string]string
+		want            map[string]string
+		wantErr         string
+
+		// fakeable errors
+		readFileErr  error
+		statErr      error
+		writeFileErr error
+	}{
```

Comment:
Let's add a test case where the same file is named in multiple different ways, like `a.txt` and `./a.txt`

Reply:
See `repeated_file_directory_only_visited_once` in `render_action_test.go`